app/views/contacts/index.html.erb

```diff
@@ -1,43 +1,119 @@
-<h1>Contacts</h1>
-
-<table>
-  <thead>
-    <tr>
-      <th>Name</th>
-      <th>Last name</th>
-      <th>Phone number</th>
-      <th>Communication channel</th>
-      <th>User</th>
-      <th>Email</th>
-      <th>Organization</th>
-      <th>Facebook</th>
-      <th>Instagram</th>
-      <th>Twitter</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @contacts.each do |contact| %>
-      <tr>
-        <td><%= contact.name %></td>
-        <td><%= contact.last_name %></td>
-        <td><%= contact.phone_number %></td>
-        <td><%= contact.communication_channel %></td>
-        <td><%= contact.user_id %></td>
-        <td><%= contact.email %></td>
-        <td><%= contact.organization %></td>
-        <td><%= contact.facebook %></td>
-        <td><%= contact.instagram %></td>
-        <td><%= contact.twitter %></td>
-        <td><%= link_to 'Show', contact %></td>
-        <td><%= link_to 'Edit', edit_contact_path(contact) %></td>
-        <td><%= link_to 'Destroy', contact, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
+<h2 class="h2 offset-1">Your contacts
+
+<!-- Trigger/Open The Modal TODO did not realize that this was not working because I did not install popper.
+will change for a bootstrap code later-->
```

Comment:
TODO's like this are a great way plan a future PR, if you're ever unsure of how much work to break out into a PR.